src/patches/HEAD/0001-Fix-UBSAN-errors-in-decompiler.patch

```diff
@@ -17,21 +16,8 @@ Subject: [PATCH 1/2] Fix UBSAN errors in decompiler
  .../Decompiler/src/decompile/cpp/slghsymbol.cc |  2 +-
  .../Decompiler/src/decompile/cpp/type.cc       |  2 +-
  .../src/decompile/unittests/testfloatemu.cc    |  2 +-
- 13 files changed, 58 insertions(+), 22 deletions(-)
+ 12 files changed, 57 insertions(+), 21 deletions(-)
 
-diff --git a/Ghidra/Features/Decompiler/src/decompile/cpp/address.cc b/Ghidra/Features/Decompiler/src/decompile/cpp/address.cc
-index 40526a88f..8a755ca82 100644
---- a/Ghidra/Features/Decompiler/src/decompile/cpp/address.cc
-+++ b/Ghidra/Features/Decompiler/src/decompile/cpp/address.cc
-@@ -666,7 +666,7 @@ uintb uintb_negate(uintb in,int4 size)
- uintb sign_extend(uintb in,int4 sizein,int4 sizeout)
- 
- {
--  intb sval = in;
-+  uintb sval = in;
```

Comment:
I had to remove the UBSAN for this function since we need to revert it entirely in the third patch.